nixos/modules/programs/localsend.nix

```
@@ -0,0 +1,27 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+with lib; let
+  cfg = config.programs.localsend;
+  firewallPort = 53317;
+in {
+  options.programs.localsend = {
+    enable = mkEnableOption (mdDoc "localsend");
+
+    openFirewall = mkOption {
+      description = mdDoc ''
+        Whether to open the firewall port ${firewallPort} for receiving files.
+      '';
+      default = false;
+      type = types.bool;
+    };
+  };
+
+  config = mkIf cfg.enable {
+    environment.systemPackages = with pkgs; [localsend];
```

```suggestion
    environment.systemPackages = with pkgs; [ localsend ];
```

Looks like you are using alejandra? If you want to format, format with `nixfmt-rfc-style`.

---

My system is not state version `24.05` yet so I cannot install it. Is using `nixfmt` acceptable?